### PR TITLE
Clean up the README file

### PR DESCRIPTION
**Pull request by Atul9**
Closes https://github.com/RustScan/RustScan/issues/101

### CODE REVIEW

**Comment by Atul9** [comment]
@brandonskerritt 
1. Can you please share which `Community link isn't working` in the readme?
2. Regarding `Move table of contents higher up` - how much higher?

File: README.md [hunk 1]
```diff
@@ -22,10 +22,10 @@
...
 | ---- | ---- | ---- | --- |
-| <p align="center"><img src="pictures/rust.png" /></p>  | <p align="center"><img src="pictures/arch.png" /></p> | <p align="center"><img src="pictures/apple.png" /></p> | <p align="center"><img src="pictures/kali.png" /></p> |
-| `cargo install rustscan` | `yay -S rustscan` | `brew tap brandonskerritt/rustscan && brew install rustscan` | [Read the install guide](https://github.com/brandonskerritt/RustScan/blob/master/README.md#%EF%B8%8F-debian--kali) |
+| <p align="center"><img src="pictures/rust.png" /></p>  | <p align="center"><img src="pictures/arch.png" /></p> | <p align="center"><img src="pictures/docker.png" /></p> | <p align="center"><img src="pictures/kali.png" /></p> |
+| `cargo install rustscan` | `yay -S rustscan` | `docker run -it rustscan:alpine` | [Read the install guide](https://github.com/brandonskerritt/RustScan/blob/master/README.md#%EF%B8%8F-debian--kali) |
```
Comment (brandonskerritt) [inline README.md:28]: My bad! The docker command is Is `docker run -it cmnatic/rustscan:debian-buster rustscan <ip>`m it must not have been mentioned in the issue :)

File: README.md [hunk 2]
```diff
@@ -64,6 +64,18 @@ RustScans **only** job is to reduce the friction between finding open ports and
 * Let's you choose what Nmap commands to run, or uses the default.
 * IPv6 Support
 
+## ‼️ Important Links
+
+| Installation Guide | Documentation | Discord |
+| ------------------ | ------------- | ------- |
+| 📖 [Installation Guide](https://github.com/RustScan/RustScan#-full-installation-guide) | 📚 [Documentation](https://github.com/RustScan/RustScan/issues/89) | 🦜 [Discord](https://discord.gg/GFrQsGy)
+
+## 🙋 Table of Contents
```
Comment (brandonskerritt) [inline README.md:73]: This is perfect placement :D

File: README.md [hunk 3]
```diff
@@ -22,10 +22,10 @@
...
+| <p align="center"><a href="https://crates.io/crates/rustscan">🔧 Cargo (Universal) </a></p> | <p align="center"><a href="https://crates.io/crates/rust_scan"> Arch </a></p> | <p align="center"><a href="https://crates.io/crates/rust_scan"> Docker </a></p> | <p align="center"><a href="https://crates.io/crates/rust_scan"> Kali / Debian </p> |
 | ---- | ---- | ---- | --- |
-| <p align="center"><img src="pictures/rust.png" /></p>  | <p align="center"><img src="pictures/arch.png" /></p> | <p align="center"><img src="pictures/apple.png" /></p> | <p align="center"><img src="pictures/kali.png" /></p> |
-| `cargo install rustscan` | `yay -S rustscan` | `brew tap brandonskerritt/rustscan && brew install rustscan` | [Read the install guide](https://github.com/brandonskerritt/RustScan/blob/master/README.md#%EF%B8%8F-debian--kali) |
+| <p align="center"><img src="pictures/rust.png" /></p>  | <p align="center"><img src="pictures/arch.png" /></p> | <p align="center"><img src="pictures/docker.png" /></p> | <p align="center"><img src="pictures/kali.png" /></p> |
```
Comment (brandonskerritt) [inline README.md:27]: Theres no `docker.png` in `pictures`/ would you mind adding one? It has to be a specific size too. I _think_ `64x64` pixels, it should look the same like all the other logos :D

Comment (Atul9) [reply README.md:27]: yes, I have added one by generating a 64x64 pixels its a part of this commit https://github.com/RustScan/RustScan/blob/c26f09b3aca569018c2cd73cc0004aef802b4e7e/pictures/docker.png

**Comment by Atul9** [comment]
> * [ ]  The community link in the table of contents is broken.

@brandonskerritt This is the table of contents, which one of them do you refer as community link? 
table-of-contents

**Comment by brandonskerritt** [comment]
> > * [ ]   The community link in the table of contents is broken.
> 
> @brandonskerritt This is the table of contents, I didn't find any community link. which one of them do you refer as community link?
> table-of-contents

Hey! Sorry, it's contributing. That's why it's not working, the name changed from contributing to community :stuck_out_tongue_closed_eyes:  https://github.com/RustScan/RustScan#-community

**Comment by brandonskerritt** [comment]
@all-contributors please add Atul9 for documentation

**Comment by brandonskerritt** [comment]
@all-contributors please add @Atul9 for documentation

**Comment by Atul9** [comment]
@brandonskerritt done, I have updated the PR

**Pull request merged**